### PR TITLE
Add some classes

### PR DESCRIPTION
**Pull request by SuperHackio**
Adds the following classes:
- RailMoveObj
- SimpleMapObj
- SimpleBreakableObj
- ActorCameraInfo

Adds the following header files:
- ActorCameraUtil

### CODE REVIEW

File: include/MapObj/RailMoveObj.h [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+	void exeMove();
+
+	TVec3f _D0; //Clipping related, used by the Init function for the MR::initAndSetRailClipping function call
+	bool isRailMoverWorking; //_D4  -Only used in exeMove, it is always set to the result of MapObjActorUtil::isRailMoverWorking. Probably used to see if the rail mover hasn't started yet
+};
+
+namespace NrvRailMoveObj {
+	class NrvWait : public Nerve
+	{
+		NrvWait() {}
```
Comment (shibbo) [inline include/MapObj/RailMoveObj.h:43]: we do not define these nerve constructors, the game does -- so remove all of the nerve constructors here

File: include/MapObj/RailMoveObj.h [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+		virtual void execute(Spine*) const;
```
Comment (shibbo) [inline include/MapObj/RailMoveObj.h:44]: space, Spine *

File: include/MapObj/RailMoveObj.h (hunk 1 above)
Comment (shibbo) [inline include/MapObj/RailMoveObj.h:43]: public identifier is also missing

File: include/MapObj/RailMoveObj.h [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+	{
+		NrvDone() {}
+		virtual void execute(Spine*) const;
+
+		static NrvDone sInstance;
+	};
+
+	class NrvDemoPlay : public Nerve
+	{
+		NrvDemoPla() {}
```
Comment (shibbo) [inline include/MapObj/RailMoveObj.h:75]: typo

File: include/MapObj/SimpleBreakableObj.h [hunk 4]
```diff
@@ -0,0 +1,79 @@
+#pragma once
+
+#include "Actor/LiveActor/LiveActor.h"
+#include "Actor/LiveActor/ActorCameraInfo.h"
```
Comment (shibbo) [inline include/MapObj/SimpleBreakableObj.h:4]: I would forward declare the class used here, as it's only referenced from a pointer

File: include/MapObj/SimpleMapObj.h [hunk 5]
```diff
@@ -0,0 +1,12 @@
+#pragma once
+
+#include "Actor/LiveActor/LiveActor.h"
```
Comment (shibbo) [inline include/MapObj/SimpleMapObj.h:3]: remove include, as MapObjActor already includes LiveActor due to inheritance

File: include/Util/ActorCameraUtil.h [hunk 6]
```diff
@@ -0,0 +1,11 @@
+#pragma once
+
+#include "kamek.h"
+
+#include "Actor/LiveActor/LiveActor.h"
```
Comment (shibbo) [inline include/Util/ActorCameraUtil.h:5]: forward declare LiveActor in utility headers

File: include/Actor/LiveActor/ActorCameraInfo.h [hunk 7]
```diff
@@ -0,0 +1,12 @@
+#pragma once
+#include "JMap/JMapInfoIter.h"
+
+class ActorCameraInfo
+{
+public:
+	ActorCameraInfo(const JMapInfoIter &);
+	ActorCameraInfo(u32, u32);
+
+	u32 CameraSetID;
```
Comment (shibbo) [inline include/Actor/LiveActor/ActorCameraInfo.h:10]: missing "m" prefix for member, and missing offset comment for both variables

File: include/MapObj/SimpleBreakableObj.h [hunk 8]
```diff
@@ -0,0 +1,78 @@
...
+	void exeDemo();
+
+	ActorCameraInfo* mActorCameraInfo; //_D0
+	u32 arg5; //_D4
+	u32 _D8;
+	u32 _DC;
+	u16 _E0;
+	u32 _E4;
+	u32 _E8;
+
```
Comment (shibbo) [inline include/MapObj/SimpleBreakableObj.h:46]: remove space

File: include/MapObj/SimpleBreakableObj.h [hunk 9]
```diff
@@ -0,0 +1,78 @@
...
+	BenefitItemObj* mOneUpKinokio; //_EC
+	u8 _F0;
+	u8 _F1;
+	u8 _F2;
+};
+
+namespace NrvSimpleBreakableObj
+{
+	class NrvWait : public Nerve
```
Comment (shibbo) [inline include/MapObj/SimpleBreakableObj.h:55]: indentation is too far right

**Comment by SuperHackio** [comment]
I hate git so much sometimes

**Pull request merged**